spec/system/articles/user_visits_an_article_spec.rb

```diff
@@ -43,6 +43,28 @@
       selector = "article time[datetime='#{timestamp}']"
       expect(page).to have_selector(selector)
     end
+
+    context "when articles have long markdowns and different published dates" do
+      let(:first_article) { build(:article, published_at: "2019-03-04T10:00:00Z") }
+      let(:second_article) { build(:article, published_at: "2019-03-05T10:00:00Z") }
+
+      before do
+        [first_article, second_article].each do |article|
+          additional_characters_for_long_markdown = (ArticlesHelper::LONG_MARKDOWN_THRESHOULD + 1) - article.body_markdown.length
+          article.body_markdown += Faker::Hipster.paragraph_by_chars(characters: additional_characters_for_long_markdown)
```

Comment:
**nitpick (non-blocking):** We can use `<<` over `+=` here, which saves us from creating a new string object:

```ruby
article.body_markdown.object_id
#=> 90500
article.body_markdown << "test"
article.body_markdown.object_id
#=> 90500
article.body_markdown += "test"
article.body_markdown.object_id
#=> 91080
```